hackathon_site/dashboard/frontend/src/api/types.ts

```diff
@@ -69,6 +69,9 @@ export interface UserWithoutProfile {
     email: string;
 }
 
+export interface slkdf extends ProfileWithoutTeamNumber {
```

Comment:
Is there a reason why you named the interface that way? lol

Reply:
Omg no I was explaining something and accidentally committed those changes 🤦‍♀️ thanks for catching